userspace/libsinsp/parsers.cpp

```diff
@@ -3273,11 +3273,11 @@ void sinsp_parser::parse_connect_exit(sinsp_evt *evt)
 	if(parinfo->m_len == 0)
 	{
 		//
-		// No address, there's nothing we can really do with this.
-		// This happens for socket types that we don't support, so we have the assertion
-		// to make sure that this is not a type of socket that we support.
+		// Address can be NULL:
+		// sk is a TCP fastopen active socket and
+		// TCP_FASTOPEN_CONNECT sockopt is set and
+		// we already have a valid cookie for this socket.
```

Comment:
So basically it can happen that there is a NULL address and the socket is unix or ipv4.

QQ, if you know: does that mean that in those cases we have bogus data, missing fields or will we already have an address?

Reply:
To be more precise:
- In ipv4 and ipv6 sockets could append that `addr = NULL` and could be expected behaviour
- In unix sockets `addr = NULL` it's not considered correct behaviour

Yes, in that case I think that the connection is already open and so we already have a correct address. 😄